src/sass/_variables.scss

```diff
@@ -10,6 +10,10 @@ $text-color: #2a2a2a;
 
 $form-border-color: #292929;
 
-$active-menu-color: #ffffff;
 
-$menu-border-top-color: #ffffff;
+$full-white: #ffffff;
+$total-white: $full-white;
+$active-menu-color: $full-white;
+$menu-border-top-color: $full-white;
+
+$grey-rgb-103:rgb(103,103,103);
```

Comment:
variable should have meaningful name that provides information what it is for, not it's content